docs/source/models/gbd2017/causes/neonatal_sepsis/index.rst

```diff
@@ -18,3 +18,137 @@ pneumonia as a separate entity.
 
 .. todo::
    Describe cause model
+   
+
+Data Description
+----------------
+
+.. list-table:: Definitions
+   :widths: 5 5 20
+   :header-rows: 1
+
+   * - State
+     - State Name
+     - Definition
+   * - S
+     - Susceptible
+     - Susceptible to neonatal sepsis and other infections
+   * - I
+     - Infected
+     - Infected with neonatal sepsis and other infections
+
+
+.. list-table:: States Data
+   :widths: 20 25 30 30
+   :header-rows: 1
+   
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - S
+     - prevalence
+     - 1-prevalence_c383
+     - 
+   * - S
+     - birth prevalence
+     - 1-birth_prevalence_c383
+     - 
+   * - S
+     - excess mortality rate
+     - 0
+     - 
+   * - S
+     - disabilty weights
+     - 0
+     -
+   * - I
+     - prevalence
+     - prevalence_c383
+     - 
+   * - I
+     - birth prevalence
+     - 1-birth_prevalence_c383
+     - 
+   * - I
+     - excess mortality rate
+     - :math:`\frac{\text{deaths_c383}}{\text{population} \times \text{prevalence_c383}}`
+     - = (cause-specific mortality rate) / prevalence
+   * - I
+     - disability weights
+     - :math:`\displaystyle{\sum_{s\in \text{sequelae_c383}}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
+     - = average disability weight over all sequelae
+   * - ALL
+     - cause specific mortality rate
+     - :math:`\frac{\text{deaths_c383}}{\text{population}}`
+     - 
+
+
+.. list-table:: Transition Data
+   :widths: 10 10 10 20 30
+   :header-rows: 1
+   
+   * - Transition
+     - Source 
+     - Sink 
+     - Value
+     - Notes
+   * - i
+     - S
+     - I
+     - :math:`\frac{\text{incidence_rate_m1594}}{\text{1 - prevalence_c383}}`
+     - Incidence rate is estimated in dismod but does not exists in como.
```

Comment:
If incidence rate comes from dismod, I think we don't need to divide through by prevalence.

Reply:
I will change it.